stable/grafana/templates/deployment.yaml

```diff
@@ -146,3 +151,15 @@ spec:
       {{- else }}
           emptyDir: {}
       {{- end -}}
+{{- if .Values.ldap.enabled }}
+        - name: ldap
+          secret:
+            {{- if .Values.ldap.createSecret }}
```

Comment:
Remove this flag. Just do this instead:

```yaml
{{- if .Values.ldap.existingSecret }}
  secretName: {{ template "grafana.fullname" . }}
{- else }}
  secretName: {{ .Values.ldap.existingSecret }}
{{- end }}
```

Reply:
I think you have that block backwards. Should be: 
```yaml
{{- if .Values.ldap.existingSecret }}
  secretName: {{ .Values.ldap.existingSecret }}
{- else }}
  secretName: {{ template "grafana.fullname" . }}
{{- end }}
```

Reply:
Addressed